libs/utils/utils.js

```diff
@@ -564,9 +564,10 @@ function decorateHeader() {
   const metadataConfig = getMetadata('breadcrumbs')?.toLowerCase()
   || getConfig().breadcrumbs;
   if (metadataConfig === 'off') return;
+  const baseBreadcrumbs = getMetadata('breadcrumbs-base')?.length;
```

Comment:
might be over engineering, but should we check whether the content is an actual URL?

Reply:
Let's trust authors for now and keep an eye on the lana logs